### ISSUE TITLE
missing title

### ISSUE THREAD

**Issue by kitten77** [guessit-io/guessit#585]
missing title when brakets are present
```Python
>>> from guessit import guessit
>>> name = '[ Engineering Catastrophes S02E10 1080p AMZN WEB-DL DD+ 2.0 x264-TrollHD ]'
>>> g = guessit(name)
>>> print(g["title"])
Traceback (most recent call last):
  File "<input>", line 1, in <module>
KeyError: 'title'
>>>print(g)
MatchesDict([('season', 2), ('episode', 10), ('screen_size', '1080p'), ('streaming_service', 'Amazon Prime'), ('source', 'Web'), ('audio_codec', 'Dolby Digital Plus'), ('audio_channels', '2.0'), ('video_codec', 'H.264'), ('release_group', 'TrollHD '), ('type', 'episode')])
(venv) muad@podde:~/workbench/gitlab/tonder$ guessit --version
+-------------------------------------------------------+
+                   GuessIt 3.0.3                       +
+-------------------------------------------------------+
+                   Rebulk 1.0.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
```

**Comment by kitten77** [comment]
workaround that fixes this 
```python
            if name[0] == '[' and name[-1] == ']':
                #added to catch bad formated strings
                #thanks deniska for coming up with this one :)
                name = name[1:-1]
```

**Comment by JohnDoee** [comment]
Wouldn't it be smarter to just trim it before handing it to guessit ?

**Comment by kitten77** [comment]
Sorry i wasn't completly clear in my workaround, updated it with all code

**Comment by ratoaq2** [comment]
Fixed by 0e1dc09aa3a73ad95a4787e84fd1e35e5445245f
